src/nvim/edit.c

```diff
@@ -4347,14 +4351,16 @@ static bool ins_tab(void)
     getvcol(curwin, cursor, &want_vcol, NULL, NULL);
 
     char *tab = "\t";
-    chartabsize_T cts;
-    init_chartabsize_arg(&cts, curwin, 0, vcol, tab, tab);
+    int32_t tab_v = (uint8_t)(*tab);
+
+    CharsizeArg arg;
+    CSType type = init_charsize_arg(&arg, curwin, 0, tab);
```

Comment:
`type` as a variable name is too general. I think it better to use `cstype` for all these variables.